src/main/java/net/levelz/gui/LevelzGui.java

```diff
@@ -75,8 +75,18 @@ public LevelzGui(MinecraftClient client) {
         WDynamicLabel fortuneLabel = new WDynamicLabel(() -> "" + BigDecimal.valueOf(playerEntity.getAttributeValue(EntityAttributes.GENERIC_LUCK)).setScale(2, RoundingMode.HALF_DOWN).floatValue());
         WDynamicLabel overallLevel = new WDynamicLabel(() -> String.format(Language.getInstance().get("text.levelz.gui.level"), playerStatsManager.getLevel("level")));
         WDynamicLabel skillPoints = new WDynamicLabel(() -> String.format(Language.getInstance().get("text.levelz.gui.points"), playerStatsManager.getLevel("points")));
-        WDynamicLabel nextLevel = new WDynamicLabel(
-                () -> "XP " + (int) (playerStatsManager.levelProgress * playerStatsManager.getNextLevelExperience()) + " / " + playerStatsManager.getNextLevelExperience());
+//        WDynamicLabel nextLevel = new WDynamicLabel(
+//                () -> "XP " + (int) (playerStatsManager.levelProgress * playerStatsManager.getNextLevelExperience()) + " / " + playerStatsManager.getNextLevelExperience());
+        WDynamicLabel nextLevel;
+        ZWButton levelUp = new ZWButton(Text.translatable("text.levelz.gui.level_up"), 10);
```

Comment:
This button should only be visible when the config setting is true

Reply:
This is working fine
![image](https://user-images.githubusercontent.com/19365612/201505485-e5f13ecb-bb0e-4ae2-ac2e-cb56897b1e89.png)